custom_components/ramses_cc/climate.py

```diff
@@ -510,13 +520,6 @@ def current_temperature(self) -> float | None:
         """Return the current temperature."""
         return self._device.indoor_temp
 
-    @property
-    def extra_state_attributes(self) -> dict[str, Any]:
```

Comment:
Note that I haven't added `params` here as the attribute wasn't previously present (RamsesHvac did not extend RamsesZoneBase).

Reply:
This is only because HVAC hasn't gotten the love it deserves - OK for now.